manifests/role/apt_mirror.pp

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2021 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::role::apt_mirror
+#
+# Debian apt mirror
+#
+# @example
+#   include nebula::role::apt_mirror
+class nebula::role::apt_mirror {
+  include nebula::role::aws
+  include nebula::profile::krb5
+  include nebula::profile::duo
```

Comment:
Is there any need for regular user logins for a one-off machine like this? I think these can go away.

Reply:
I guess I need clarification on this. Will remove those two still allow us to log in as us? Our accounts are created separate right?

Reply:
Also done.